cmd/root.go

```diff
@@ -17,6 +17,18 @@ func cmd() *cobra.Command {
 	viper.SetDefault("deadline", 18000)
 
 	viper.SetConfigName("config")
+
+	// Check if CTL version is the latest version by comparing local CTL version with remote version tags from GitHub
+	var WarningColor = "\033[1;33m%s\033[0m"
+	out, err := exec.Command("bash", "-c", "git ls-remote --tags https://github.com/wish/ctl.git | tail -n 1 | cut -d'/' -f3 |  cut -d'^' -f1").Output()
+    if err != nil {
+		fmt.Printf("Unable to retrieve remote CTL tags for version comparison. Error: %s\n",err)
+		os.Exit(1)
```

Comment:
I would probably change it so that if for some reason we can't get the tags it doesn't exit. In the worse case scenario, we want users to be able to use the tool even if its not the latest version rather than blocking them if git fails for some reason

Reply:
Updated to remove Exit command to let users utilize it despite git errors